src/entry-point-client/PageList.tsx

```diff
@@ -2,6 +2,7 @@ import GetInvolvedPage from "../page-get-involved/GetInvolvedPage";
 import OverviewPage from "../page-overview/OverviewPage";
 import UnsupportedBrowserPage from "../page-unsupported-browser/UnsupportedBrowserPage";
 import VerificationNeeded from "../page-verification-needed/VerificationNeeded";
+import MultiFacilityPage from "../page-multi-facility/MultiFacilityPage";
```

Comment:
Sorry this isn't explicit in the readme, @jovergaag! You should run `yarn lint-all-check` or `yarn lint-all-fix` locally before pushing up a commit, to clear out any linting issues. It would catch this out of order import, for example.

Reply:
I just saw that failure in the CI. Its rebuilding now with the fix. Thanks!